photon-ml/src/main/scala/com/linkedin/photon/ml/io/InputFormatType.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.io
+
+/**
+  * @author yitong_zhou
+  */
```

Comment:
Please add some short documentation about this Enum (I know it's pretty obvious from the name but we should be consistent)
We've also decided to get rid of author tags (it's happening slowly as files that still contain them get touched)